src/services/gateway.service.ts

```diff
@@ -317,6 +318,43 @@ function removeThemeLogo(
   )
 }
 
+type DelegateResponse = Omit<Delegate & { created_at: string }, 'createdDate'>
+
+async function fetchAppDelegates(
+  appId: AppId
+): Promise<AxiosResponse<DelegateResponse[]>> {
```

Comment:
If you are getting a TypeScript error here because for the `Promise<...>` return type, try:

```ts
return Promise.resolve({ status: 200, ... })
```

Reply:
Resolved here https://github.com/arcana-network/developer-dashboard/pull/144/commits/2bad363f2d40991a4be2e50b7001832cdef39536